microsoft-edge/webview2/code-samples-links.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Links to WebView2 sample code hosted on GitHub 
+title: WebView2 sample code
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, source code, sample code, samples, code samples, WebView sample code, WebView2 sample code
+---
+# WebView2 sample code
```

Comment:
Global: Try h3s instead of verbose bullet list.  To make skimmable (as written in MWSG).

Reply:
done